template/web/gatsby-node.js

```diff
@@ -46,6 +44,70 @@ async function createBlogPostPages (graphql, actions) {
     })
 }
 
+async function createCategoryPages (graphql, actions) {
+  const {createPage} = actions
+
+  const result = await graphql(`
+    {
+      allSanityCategory {
+        edges {
+          node {
+            id
+            slug {
+              current
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  if (result.errors) throw result.errors
+
+  const categoryNodes = (result.data.allSanityCategory || {}).edges || []
+
+  categoryNodes.forEach(node => {
+    const {id, slug} = node.node
+    if (!id || !slug.current) return 'No category id or slug!'
```

Comment:
Prefer braces `{}` for conditional statements. Are you sure we should return anything here btw? Is it better to use the Gatsby reporter for this?

Reply:
yep, `reporter.warn` is most suitable here (it's not an error per se if there are no categories, yet it's something to pay attention to because it's likely that you forgot to update `Category` schema with a `slug`)